.log.io/harvester.conf

```diff
@@ -1,7 +1,5 @@
 var config = {};
 config = Object.assign(config, require('../config.json'));
-try {
-  config = Object.assign(config, require('../config.local.json'));
-} catch (err) {}
+config = Object.assign(config, require('../config.local.json'));
```

Comment:
if config.local.json do not exists, it will crash if u remove the try catch, no?